install/linux/usr/share/odemis/sim/meteor-sim.odm.yaml

```diff
@@ -71,14 +71,17 @@
         FM_IMAGING_RANGE: {"x": [0.040, 0.054], "y": [-5.e-3, 10.e-3], "z": [-0.5e-3, 8.e-3]},
         # Grid centers in SEM range
         # Adjusted so that at init (0,0,0), it's at the Grid 1.
-        SAMPLE_CENTERS: {"GRID 1": {'x': 0, 'y': 0, 'z': 0}, "GRID 2": {'x': 2.98e-3, 'y': 2.46e-3, 'z': 0}},
+        SAMPLE_CENTERS: {"GRID 1": {'x': 0, 'y': 0, 'z': 0}, "GRID 2": {'x': 5.0e-3, 'y': 0, 'z': 0}},
         # Mirroring values between SEM - METEOR
-        POS_COR: [0.02447, -0.000017],
+        POS_COR: [0.02447, -0.000017],  # [m] rotation correction (x, y) (sem <-> meteor)
+        POS_TRANS_COR: [48.8e-3, 0],    # [m] translation correction (x, y) (fib <-> meteor), standard value is x=48.8mm
+        SAMPLE_PRE_TILT: 0.61,          # [rad] pre-tilt angle for the sample holder, standard value is 35°
         # Active tilting (rx) & rotation (rz) angles positions when switching between SEM & FM, in radians.
         # Note: these values are calibrated at installation time.
-        FAV_FM_POS_ACTIVE: {"rx": 0.12213888553625313  , "rz":  5.06145}, # 7° - 270°
-        # Typically rz = 110°, but we make it 0 so that at init it looks like in SEM position
-        FAV_SEM_POS_ACTIVE: {"rx": 0, "rz": 0}    # Note that milling angle (rx) can be changed per session 
+        FAV_FM_POS_ACTIVE: {"rx": 0.2968  , "rz":  3.1415}, # 17° - 180°, FM position (perpendicular to the fm)
+        FAV_FIB_POS_ACTIVE: {"rx": 0.2968, "rz": 3.1415},   # 17° - 180°, FIB position (perpendicular to the fib)
+        FAV_SEM_POS_ACTIVE: {"rx": 0.61, "rz": 0},          # 35° - 0° sem position (perpendicular to the sem)
```

Comment:
Have 4 decimal place precision like above? similarly for `FAV_MILL_POS_ACTIVE`, `rotation`